### ISSUE TITLE
Doesn't work with OSP...

### ISSUE THREAD

**Issue by iradicus** [ry4096/znudge#1]
Very interesting man. I just tried it to see what it was like. Pretty unique with the ball of light. I noticed that OSP settings are gone on servers that support it (cg_enemymodel, cg_enemycolors, etc.). Is it possible to have them play nice with each other? Or at least when you turn off znudge 0 so you can toggle between them without having to completely uninstall znudge? -Wizzy

**Comment by ry4096** [comment]
Yeah, the right way to do this would be to get the source code for osp and to merge this mod into the osp's cgame. But since I don't have that I think the only feasible option would be to reinvent the osp cgame features in an open source mod. Some features are easier than others and I could just add them them in.

**Comment by iradicus** [comment]
It's a real shame that the author never published the source to OSP. I'm sure you already have the OSP documentation, but I'm attaching the Client Readme which I think covers everything...
[osp-q3-ClientReadme.txt](https://github.com/ry4096/znudge/files/1741421/osp-q3-ClientReadme.txt)
